.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,42 @@
+name: "[Automation] Pull Request"
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened, closed]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: anencore94/labeler@v1.1.0
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+  assign-reviewer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: reviewers
+        uses: hkusu/review-assign-action@v1
+        with:
+          assignees: ${{ github.actor }}
+          reviewers: dongminlee94, limjiyoon, yurirocha15
+          max-num-of-reviewers: 2
+          draft-keyword: WIP
```

Comment:
What about using CODEOWNERS instead of this action?

Reply:
Done.